src/components/Order/Disclaimer.tsx

```diff
@@ -1,20 +1,25 @@
 import * as React from 'react';
+import {disclaimerMessages} from '../../constants/assetDisclaimer';
 import {Body, DisclaimerNotification, Link, Title} from './styles';
 
-export const Disclaimer = () => (
-  <DisclaimerNotification>
-    <Title>Attention</Title>
-    <Body>
-      <p>
-        Due to the EOS migration, deposits and withdrawals are disabled until
-        further notice.
-      </p>
-      <Link
-        target="_blank"
-        href="https://www.lykke.com/company/news/2018-05-eos-mainnet-swap"
-      >
-        Read more
-      </Link>
-    </Body>
-  </DisclaimerNotification>
-);
+interface DisclaimerProps {
+  asset: string;
+}
+
+export const Disclaimer: React.SFC<DisclaimerProps> = ({asset}) => {
+  const content = disclaimerMessages[asset];
```

Comment:
Maybe it make sense to check that content for asset is presented? Just to avoid rendering incorrect content